### ISSUE TITLE
Improve behaviour

### ISSUE THREAD

**Issue by huyphams** [robin/LYTabView#5]
Hi Robin,

I have an issue when using TabView. Here is my gif.
![issue](https://cloud.githubusercontent.com/assets/3654622/25372271/f4f2bfb4-29be-11e7-8baf-13af186fd514.gif)

I removed this line: 
```

        set(newTitle) {
            titleView.stringValue = newTitle as String
//            self.invalidateIntrinsicContentSize()
        }

```
https://github.com/robin/LYTabView/blob/master/LYTabView/LYTabItemView.swift#L70
And it much better.

An other improve is set titleView breakmode to
```
        titleView.lineBreakMode = .byTruncatingMiddle
```
https://github.com/robin/LYTabView/blob/master/LYTabView/LYTabItemView.swift#L116

**Comment by robin** [comment]
It's a compression resistance priority issue indeed. I've fixed it in the latest v0.17. Can you have a try?

**Comment by huyphams** [comment]
It's fixed 👍🏻
